action/protocol/execution/evm/evmstatedbadapter.go

```diff
@@ -136,14 +135,6 @@ func RevertLogOption() StateDBAdapterOption {
 	}
 }
 
-// CheckPutStateErrorOption set checkPutStateError as true
-func CheckPutStateErrorOption() StateDBAdapterOption {
-	return func(adapter *StateDBAdapter) error {
-		adapter.checkPutStateError = true
-		return nil
-	}
-}
-
```

Comment:
why revert, the comment is to say flip this bool flag, so it == false for after hard-fork

Reply:
sorry, misunderstand.  I modify as notCheckPutStateError